src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -121,11 +124,15 @@ private void bindDrivebaseControls() {
 	private void bindIntakeControls() {
 		// CommandScheduler.getInstance()
 		// 		.setDefaultCommand(s.intakeSubsystem, new IntakeStopCommand(s.intakeSubsystem));
-		driveIntakeInButton.onTrue(new AllInCommand(s.intakeSubsystem));
+		driveIntakeInButton.onTrue(
+				Commands.sequence(
+						new AllInCommand(s.intakeSubsystem, this),
+						new WaitCommand(0.5),
+						new StopRumbleCommand(this)));
```

Comment:
I'm pretty sure the controller would get stuck vibrating if the driver or codriver pressed any of the other intake buttons right after a note is detected & the controller starts rumbling. That would interrupt this sequence during the WaitCommand so the StopRumbleCommand would never run. One way to fix that is to wrap AllInCommand and WaitCommand in a new command class derived from SequentialCommandGroup and have the `end(..)` method call controls.vibrateDriveController(0.0).